deployer/src/helpers.py

```diff
@@ -46,61 +41,64 @@ def make_request(endpoint, type=None, data=None, username=None, password=None,
                  json_request=False):
     import requests
 
-    url = base_url + endpoint if "://" not in endpoint else endpoint
+    if "://" not in endpoint:
+        print("Wrong endpoint:{}, it should be a complete URL".format(endpoint))
```

Comment:
I am unsure: validating a URL is close to impossible. I mean, we could check for the syntax of course (<schema> :// <domain> / etc), then check for the theoretical validity (eg schema is "http" or "https", and so on), but ultimately if the host is not there or not up or if the path is incorrect, it will fail. All of this is done while requesting.

= Do we need this check? It really seems way too light to me - it is close to useless, and would remove it (eg if endoint is "://://://" it will pass; if endoint is "foobar://" it will too). If we want to check the validity of configs before starting (we most probably do), let's do that in another set of commits, introducing a specific tool/feature for this, which it would do at least a complete syntax check and ensure that the protocols are the ones supported.

Reply:
Please answer my question, and let me close conversations; I cannot tell if you have read/understood/answered the point otherwise